.eslintrc.json

```diff
@@ -1,8 +1,39 @@
 {
-  "plugins": ["import"],
-  "extends": ["react-app", "react-app/jest", "prettier"],
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier"
+  ],
   "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "plugins": ["react", "@typescript-eslint", "import", "react-hooks"],
   "rules": {
+    "@typescript-eslint/no-non-null-assertion": "off",
```

Comment:
keep for now

Reply:
sounds good - we shoudl make issues for these if we haven't already

Reply:
Totttalllyy. Great call. done for all relevant ones.

Follow-up for this one Follow-up issue https://github.com/jbx-protocol/juice-interface/issues/1139